servers/rendering/shader_language.cpp

```diff
@@ -3312,12 +3316,18 @@ bool ShaderLanguage::_validate_varying_assign(ShaderNode::Varying &p_varying, St
 				p_varying.stage = ShaderNode::Varying::STAGE_FRAGMENT;
 			}
 			break;
+		case ShaderNode::Varying::STAGE_VERTEX_TO_FRAGMENT:
+			[[fallthrough]];
+		case ShaderNode::Varying::STAGE_VERTEX_TO_LIGHT:
+			[[fallthrough]];
 		case ShaderNode::Varying::STAGE_VERTEX:
```

Comment:
If that's a style you already use in other places to be explicit that's fine, but note that this would be equivalent too, the fallthrough is implicit when there's no code:
```suggestion
		case ShaderNode::Varying::STAGE_VERTEX_TO_FRAGMENT:
		case ShaderNode::Varying::STAGE_VERTEX_TO_LIGHT:
		case ShaderNode::Varying::STAGE_VERTEX:
```

Reply:
Ok. I didn't memorize all of this stuff, sorry.